driver-core/src/main/java/com/datastax/driver/core/querybuilder/Insert.java

```diff
@@ -56,7 +56,7 @@ StringBuilder buildQueryString(List<Object> variables) {
         if (keyspace != null)
             Utils.appendName(keyspace, builder).append('.');
         Utils.appendName(table, builder);
-        builder.append('(');
+        builder.append(" (");
```

Comment:
Agreed, that's also how insert statements are formatted in the CQL reference doc.
IMO this should stay as a separate commit when we eventually merge the PR.